extension/src/state_aggregate.rs

```diff
@@ -671,6 +671,30 @@ fn state_agg_int_trans(
     }
 }
 
+/// Makes an interval from an `i64` representing the number of microseconds.
+fn make_interval(time: i64) -> crate::raw::Interval {
```

Comment:
We actually have a `impl From<i64> for Interval` already in `raw.rs` that does most of this already.  It seems cleaner to just add a `justify_interval` there as well and use something like `justify_interval(time.into())` instead of this function.

Reply:
That works.  I was a little reluctant to suggest this as I'm guessing the justification isn't sticky...if we add two justified intervals I don't think the result is guaranteed to be justified.  However, I'm pretty sure we never actually do math on these after we convert them to intervals, so that shouldn't be a problem.